src/3d/qgs3dmodelcache.h

```diff
@@ -0,0 +1,112 @@
+/***************************************************************************
+                         qgs3dmodelcache.h
+                         ---------------
+    begin                : July 2020
+    copyright            : (C) 2020 by Mathieu Pellerin
+    email                : nirvn dot asia at gmail dot com
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef QGS3DMODELCACHE_H
+#define QGS3DMODELCACHE_H
+
+#include "qgis_3d.h"
+#include "qgsabstractcontentcache.h"
+
+#include <Qt3DRender/QSceneLoader>
+#include <Qt3DRender/QMesh>
+
+
+///@cond PRIVATE
+
+/**
+ * \ingroup 3d
+ * \class Qgs3DModelCacheEntry
+ * An entry for a Qgs3DModelCache, representing a 3D model filename
+ * \since QGIS 3.6
```

Comment:
```suggestion
 * \since QGIS 3.16
```

Reply:
No interest in backporting to RIP 3.6 branch? :wink: